### ISSUE TITLE
Wiki-links broken after backup restore

### ISSUE THREAD

**Issue by paulkernstock** [taigaio/taiga-back#966]
I succesfully restored a taiga-backup I did using a database-dump and a /media-backup (s. #961 for details) and the system seems to have full functionality expect one thing I experienced:

On the Wiki-pane, although all wiki-sites are available, all links are broken, i.e. non-clickable markup-code is shown:

[[Link-text]]

**Comment by jespino** [comment]
Yes, this links ins't supported yet in the new editor, we have an issue opened with this problem.

**Comment by paulkernstock** [comment]
Any new insights on this issue?
